proposals/gen-two-tier-approach.md

```diff
@@ -0,0 +1,220 @@
+A Generalizable Two-Tier Voluntary Security Attestation Guideline
+===
+
+_**Editor's Notes**_:
+- We have adopted "VSA" as an acronym for "voluntary security attestation"
+  to provide disambiguation from other technical meanings of "attestation"
+  commonplace in our industry today (e.g., Remote Attestation, Hardware 
+  based Attestations, and RSAA-compliance Attestations in U.S. Federal
+  procurement).
+- _**TODO:**_ add a definition/citation for "Project."
+- _**TODO:**_ complete various TODO's throughout this draft.
+
+
+Edit History
+---
+
+| Revision | Date | Summary of Changes |
+| --- | ---------- | --- |
+| 0.1 | 18.11.2025 | Initial proposal by Æva Black to the ORC WG |
+
+
+
+Overall Recommendations
+---
+1. Stewards should produce VSAs only for projects to which they offer 
+   "sustained and ongoing support." 
+1. Stewards may produce VSAs for their project's dependencies; if done, the
+   Steward should clearly document the relationship between the subject of the
+   VSA and the project which the Steward supports.
+1. Stewards should not produce VSAs for projects that are not associated with
+   the Steward's activities.
+1. Stewards may publish VSAs in multiple ways, including:
+   1. By placing a `COMPLIANCE.MD` file in the project's root directory (i.e.,
+      alongside the `LICNESE.MD` file)
+   1. By providing a digitally-signed file to members of the non-profit legal
+      entity.
```

Comment:
As currently written, it is unclear to me what 'members' and 'the non-profit legal entity' refer to.